mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -220,14 +220,11 @@ public MessageUid findFirstUnseenMessageUid(Mailbox mailbox) throws MailboxExcep
     public Map<MessageUid, MessageMetaData> expungeMarkedForDeletionInMailbox(Mailbox mailbox, MessageRange messageRange) throws MailboxException {
         CassandraId mailboxId = (CassandraId) mailbox.getMailboxId();
 
-        return FluentFutureStream.of(deletedMessageDAO.retrieveDeletedMessage(mailboxId, messageRange))
-            .completableFuture()
-            .thenApply(stream -> stream.collect(JamesCollectors.chunker(EXPUNGE_BATCH_SIZE)))
-            .thenCompose(chunkedExpungedUids ->
-                CompletableFutureUtil.chainAll(chunkedExpungedUids,
-                    uidChunk ->  expungeUidChunk(mailboxId, uidChunk)))
-            .thenApply(s -> s.flatMap(i -> i))
+        return deletedMessageDAO.retrieveDeletedMessage(mailboxId, messageRange)
```

Comment:
The way I understand it is *that we may not read Cassandra result fast enough, thus generating  a timeout* even if that explanation seems strange to me...